integration/setup/teardown.go

```diff
@@ -19,10 +19,10 @@ func Teardown(c Config) error {
 	// Delete AzureConfig from control plane to avoid reconciling while deleting the resources on Azure.
 	// We don't wait for the resources to be deleted, since we are going to delete them below.
 	err = c.Host.G8sClient().ProviderV1alpha1().AzureConfigs(c.Host.TargetNamespace()).Delete(clusterID, &metav1.DeleteOptions{})
-	if err != nil {
-		if !errors.IsNotFound(err) {
-			return microerror.Mask(err)
-		}
+	if errors.IsNotFound(err) {
+		// Fallthrough. This is fine.
```

Comment:
Do we still need to delete resource on Azure in this case? Just curious.

Reply:
Good point. I guess we don't. I'll return nil if the `CustomResource` is not found.